backend/banners/banners.go

```diff
@@ -43,8 +44,12 @@ type Message struct {
 	ID string `json:"id"`
 	// Link, if present, will be appended to the message.
 	Link *struct {
-		Href string `json:"href"`
+		Href string  `json:"href"`
+		Text *string `json:"text"`
 	} `json:"link"`
+	// Dismissible: if true the ID field will be the key of config.frontend map to keep
+	// trace of dismissed banners (see 	status.tsx for details).
```

Comment:
there is a tab after `see` :)

Reply:
sry, my vim config has still some issues with tabs and spaces :sweat_smile: fixed